python/tvm/relay/quantize/_annotate.py

```diff
@@ -191,7 +191,7 @@ def check_to_skip():
     return False
 
 
-@register_annotate_function("nn.dense")
+# @register_annotate_function("nn.dense")
```

Comment:
You can use `debug_enabled_ops` if you want to temporarily disable this op

Reply:
Thanks for the catch. Upon discussion with @ziheng we have concluded that we need a better fix that using `debug_enabled_ops` for VTA quantization. For now instead of adding another `skip_conv` option, we'll leave this commented until we can effectively offload the dense layer to VTA. This will require operator support which will be followed up with shortly. Until then I recommend that we leave the registration line commented out, with a TODO that I'll address in the near future.

Reply:
`debug_enabled_ops` is not suitable for this case, which is a white-list for enabled operations for debugging. It means you need to list all other op except `dense` in this condition.

Reply:
ok I agree to comment it out as long as it can be fixed soon